meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=1', 'cpp_std=c++14'],
```

Comment:
The current requirement is C++11 until #95 is merged (and the doc is updated accordingly).

Reply:
I'll remove for now. Note that cpp_std=c++11 breaks on MSVC as it does not support c++11 for its std parameter.